src/Akeneo/Connectivity/Connection/back/Domain/Apps/Persistence/Query/CreateConnectedAppQueryInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Domain\Apps\Persistence\Query;
```

Comment:
```suggestion
namespace Akeneo\Connectivity\Connection\Domain\Apps\Persistence;
```
I'm wondering if it's an opportunity to update namespaces for our queries and remove needless indirections